api/include/opentelemetry/metrics/meter.h

```diff
@@ -54,7 +54,7 @@ class Meter
    * @param description a brief description of what the Observable Counter is used for.
    * @param unit the unit of metric values following https://unitsofmeasure.org/ucum.html.
    */
-  virtual nostd::shared_ptr<ObservableInstrument> CreateLongObservableCounter(
+  virtual nostd::shared_ptr<ObservableInstrument> CreateUInt64ObservableCounter(
```

Comment:
Hate to do this, but Observable Counter can be negative also, so this should be `CreateInt64ObservableCounter` :(. I confirmed offline with specs author. Sorry about multiple iterations here, but this is important to do it right before GA.

Reply:
thanks, yes it's important we leave no changes needed after GA :)
changed to `CreateInt64ObservableCounter`